libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Bot.Connector.Authentication
     /// <summary>
     /// MicrosoftAppCredentials auth implementation and cache.
     /// </summary>
-    public class MicrosoftAppCredentials : ServiceClientCredentials
+    public class MicrosoftAppCredentials : AppCredentials
```

Comment:
From an OO perspective, this is certainly legit. I think it as as well from a .NET compat perspetive.